eslintconfig/.eslintrc-server

```diff
@@ -1,4 +1,8 @@
 ---
 extends:
-  - "defaults/configurations/walmart/es5-node"
+  - "defaults/configurations/walmart/es6-node"
   - ".eslintrc-base"
+
+rules:
+  strict: [0]
```

Comment:
Without this, eslint complains that `'use strict'` is unnecessary--that's true in ES6 modules (strict by default), but we're not using ES6 modules yet.